pages/about.html

```diff
@@ -1,12 +1,55 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="preconnect" href="https://fonts.googleapis.com"> 
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+        <link rel="preconnect" href="https://fonts.googleapis.com"> 
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+        <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;600&family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+        <script src="https://kit.fontawesome.com/c72358b43f.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="/styles/index-style.css">
+
+        <title>Tasha Chang About</title>
+    </head>
+    <body>
+        <main class ="main-container">
+            <header>
+                <h1>Tasha Chang</h1>
+                <nav>
+                    <ul class="navlinks">
+                        <li><a href="../index.html">Home</a></li>
```

Comment:
This link is broken. =( After moving the index.html page out of the "pages" directory, these needed to be updated. It should be `../../index.html`. This is true for the Portfolio page as well.